schemas/compat-data-schema.md

```diff
@@ -218,12 +218,14 @@ entirely unknown. Examples:
  "version_added": "3.5"
 }
 ```
-* Supported, but version unknown:
+* Supported, but earliest version unknown:
+  The version in the version range is the version at the time of commit.
```

Comment:
I think the wording on this line is a bit confusing, and potentially inaccurate.  I think that it's more fitting to say "the version specified should be the earliest version was noticed in.  This is usually the current version of the browser."